homework_1_collections/Nucleo.py

```diff
@@ -0,0 +1,135 @@
+# Transcribes DNA to RNA.
+# dna_seq - string, DNA sequence.
+def transcribe(dna_seq):
+    x = list(dna_seq)
+    for i in range(len(x)):
+        if x[i] == 'T':
+            x[i] = 'U'
+        if x[i] == 't':
+            x[i] = 'u'
+    x = (''.join(x))

```

Comment:
Скобки не нужны и `x` - плохое название для переменной
```suggestion
    dna_list = list(dna_seq)
    for i in range(len(x)):
        if dna_list[i] == 'T':
            dna_list[i] = 'U'
        if dna_list[i] == 't':
            dna_list[i] = 'u'
    rna_seq = ''.join(dna_list)

```